unc_katavasia_mid-pentecost.byzx

```diff
@@ -0,0 +1,3098 @@
+{
+  "version": "1.0",
+  "appVersion": "0.4.11",
```

Comment:
You'll want to upgrade to the [latest release](https://github.com/neanes/neanes/releases/tag/v0.4.20) of the software, as there have been minor changes to the file format since version 0.4.11.